src/main/java/Duke.java

```diff
@@ -1,10 +1,603 @@
+        return taskNum;
+    }
+
+    /***
+     * Prior to adding events with command deadline to the list, processDeadline separates the
+     * important details within the input for data storage.
+     * @param storedValues List of tasks from user inputs.
+     * @param taskNum The existing position in the list.
+     * @param line User input to be processed.
+     * @return Updated position in the list.
+     */
+    private static int processDeadline(ArrayList<Task> storedValues, int taskNum, String line) {
+        int forwardSlash = line.indexOf('/');
+        int endOfTask = forwardSlash - 1;
+        int dates = forwardSlash + REMOVE_BY_NUM;
+        Deadline deadlineInput = new Deadline(line.substring(9, endOfTask), line.substring(dates));
+        taskNum = addTask(storedValues, taskNum, deadlineInput);
+        return taskNum;
+    }
+
+    /***
+     * Adds processed user inputs into the list of stored values.
+     * @param storedValues List of tasks from user inputs.
+     * @param taskNum The existing position in the list.
+     * @param line User input to be processed.
+     * @return Updated position in the list.
+     */
+    private static int addTask(ArrayList<Task> storedValues, int taskNum, Task line) {
+        formattingLine();
+        System.out.println("Got it. I've added this task: \n" + line.toString() + "\n" +
+                "Now you have " + (taskNum + 1) + " tasks in the list.\n");
+        formattingLine();
+        // Store value in line into list
+        storedValues.add(taskNum,line);
+        taskNum += 1;
+        try {
+            writeToFile(line);
+        } catch (IOException e) {
+            System.out.println("Something went wrong..." + e.getMessage());
+        }
+        return taskNum;
+    }
+
+    /***
+     * Unmarks the task in the list when called.
+     * @param storedValues List of tasks from user inputs.
+     * @param line User input to be processed.
+     * @throws UnmarkOutOfBounds Thrown when the value input is out of bounds.
+     * @throws IOException Thrown when other errors are detected.
+     */
+    private static void unmarkItem(ArrayList<Task> storedValues, String line) throws UnmarkOutOfBounds, IOException {
+        int length = line.length();
+        String itemToMark = line.substring(REMOVE_UNMARK_NUM, length);
+        int numToMark = Integer.parseInt(itemToMark);
+        // Unmark the item
+        if (taskNum < numToMark) {
+            throw new UnmarkOutOfBounds();
+        } else {
+            unmarkTaskInTxt(storedValues, numToMark);
+        }
+    }
+
+    /***
+     * Unmarks the tast in the txt file when called.
+     * @param storedValues List of tasks from user inputs.
+     * @param numToMark The row in txt file to be unmarked.
+     * @throws IOException Thrown when file cannot be read.
+     */
+    private static void unmarkTaskInTxt(ArrayList<Task> storedValues, int numToMark) throws IOException {
```

Comment:
This is a long method doing many things. Try having separate method handling different tasks. Helps with debugging too :)